components/ScheduleTable.tsx

```diff
@@ -85,7 +91,8 @@ export default function ScheduleTable({
                 </Flex>
               </Td>
               <Td>
-                <Text>{event.title}</Text>
+                <Text fontSize="xl">{event.title}</Text>
```

Comment:
> I think it looks a bit squished on mobile

I think so too @gordonpn

Maybe try with small or medium font size with bold font weight, and some margin on the bottom if a description is provided!